manila/share/drivers/netapp/dataontap/cluster_mode/lib_multi_svm.py

```diff
@@ -1455,9 +1447,15 @@ def _migration_start_using_svm_migrate(
 
         # 3. Send the migration request to ONTAP.
         try:
+            aggr_names = dest_share_server.get('volume_placement')
+            if aggr_names:
```

Comment:
right, nice catch. change to set aggr_names to `None` if pool is None. Also use `dest_share_server['volume_placement']` instead of using get function, assuming `volume_placement` is always set, otherwise threw key error exception.